qa/workunits/fs/full/subvolume_clone.sh

```diff
@@ -19,7 +19,6 @@ set -ex
 # Note that the failed clones would be in retry loop and it's state would be 'pending' or 'in-progress'.
 # It's state is not updated to 'failed' as the config update to gets ENOSPACE too.
 
-set -e
```

Comment:
We don't want to set to `-ex` in `subvolume_clone.sh`?

Reply:
There is a `set -ex` in line#2 of this file already.

Reply:
ah, okay. i saw every other place you added `ex` removing `e`, so thought it should be here too.